nss-day-cohort-22/Class-Website
# Updated JSON and added tech images

## Description of Proposed Changes
1. Updated the JSON file with more info from the Google Sheets document
2. Added the images for the techs
3. Linked the email/github/portfolio/linkedin images properly
4. Created placeholders for the group photos
5. Added NSS Logo at top of screen

## Steps to Test
1. `git fetch --all`
2. `git checkout kn-json`
3. If you've previously pulled down this branch, run `git pull origin kn-json` to get the updates
4. `http-server`
5. Check to make sure placeholder images are there and that the tech images appear properly
